### PR TITLE
Update version, build number, add UMF component

### PR DESCRIPTION
**Pull request by oleksandr-pavlyk**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by oleksandr-pavlyk** [comment]
@conda-forge-admin, please rerender

**Review by napetrov — APPROVED** [review]

**Comment by oleksandr-pavlyk** [comment]
@conda-forge/core For some reason CI jobs did not run on this PR. Is there anything I need to do to trigger those?

**Comment by oleksandr-pavlyk** [comment]
I suspect this is because the commit message in the last merge commit in the main branch contains `[skip ci]`. @conda-forge/core  what is the best way to fix it?

**Comment by h-vetinari** [comment]
You're running into https://github.com/conda-forge/status/issues/188, which is a problem with azure itself. Should hopefully be fixed in a few hours

**Comment by ZzEeKkAa** [comment]
@conda-forge-admin, please rerender

**Comment by ZzEeKkAa** [comment]
```
(base) %SRC_DIR%>conda activate "%PREFIX%" 
# Copyright (c) 2010 Intel Corporation. All rights reserved.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   956  100   956    0     0  36019      0 --:--:-- --:--:-- --:--:-- 38240
SYCL CPU RT Warning: Cannot load TBB from neither Windows registry key nor CPU runtime configuration file (cl.cfg / cl.fpga_emu.cfg) in C:\bld\intel-compiler-repack_1730518262540\_test_env\Library\bin\intel-ocl-cpu\ location. The Error message is: Windows error code: 126.
You can ask your administrator to configure TBB library location to CL_CONFIG_TBB_DLL_PATH item in the configuration files.
Or you need to check Windows registry key under HKEY_LOCAL_MACHINE\SOFTWARE\Intel\oneAPI\TBB\ location. The version items under this location are installed TBB on this machine. The required TBB version is 2022.0.0. You can install the required TBB if it is not listed in Windows registry.
Traceback (most recent call last):
  File "C:\bld\intel-compiler-repack_1730518262540\test_tmp\demo.py", line 9, in <module>
    ctx = cl.create_some_context()
  File "C:\bld\intel-compiler-repack_1730518262540\_test_env\Lib\site-packages\pyopencl\__init__.py", line 1701, in create_some_context
    return Context(devices)
pyopencl._cl.RuntimeError: Context failed: OUT_OF_HOST_MEMORY
WARNING: Tests failed for intel-opencl-rt-2025.0.0-h27e31e5_1162.conda - moving package to C:\bld\broken
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -513,6 +523,40 @@ outputs:
...
+      host:
+        - libhwloc
+      run:
+        # intel-cmplr-lic-rt is a license file, please do not remove.
+        - {{ pin_subpackage('intel-cmplr-lic-rt', exact=True) }}
+    about:
+      home: https://software.intel.com/content/www/us/en/develop/tools.html
+      doc_url: https://software.intel.com/content/www/us/en/develop/articles/end-user-license-agreement.html
+      dev_url: https://software.intel.com/content/www/us/en/develop/articles/end-user-license-agreement.html
+      summary: Intel End User License Agreement for Developer Tools
```
Comment (ZzEeKkAa) [inline recipe/meta.yaml:545]: We need to update summary

**Review by ZzEeKkAa — APPROVED** [review]
LGTM!

**Comment by ZzEeKkAa** [comment]
These are the only warnings left, which is okay:
```
WARNING (intel-cmplr-lib-rt): run-exports library package local/linux-64::intel-cmplr-lic-rt==2025.0.0=ha770c72_1169 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-cmplr-lib-rt): run-exports library package conda-forge/linux-64::libstdcxx-ng==14.2.0=h4852527_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (umf): run-exports library package local/linux-64::intel-cmplr-lic-rt==2025.0.0=ha770c72_1169 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-cmplr-lib-ur,lib/libur_adapter_level_zero.so.0.10.8): Needed DSO lib/libze_loader.so.1 found in ['conda-forge/linux-64::level-zero==1.18.5=h84d6215_0']
WARNING (intel-cmplr-lib-ur,lib/libur_adapter_level_zero.so.0.10.8): .. but ['conda-forge/linux-64::level-zero==1.18.5=h84d6215_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
WARNING (intel-cmplr-lib-ur): run-exports library package conda-forge/linux-64::libstdcxx-ng==14.2.0=h4852527_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-fortran-rt): run-exports library package conda-forge/linux-64::_openmp_mutex==4.5=2_gnu in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-opencl-rt): run-exports library package conda-forge/linux-64::ocl-icd==2.3.2=hd590300_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-opencl-rt): run-exports library package conda-forge/linux-64::libxml2==2.13.4=h064dc61_2 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-opencl-rt): dso library package conda-forge/linux-64::tbb==2022.0.0=hceb3a55_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-opencl-rt): run-exports library package conda-forge/linux-64::libzlib==1.3.1=hb9d3cd8_2 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (intel-sycl-rt): run-exports library package conda-forge/linux-64::libstdcxx-ng==14.2.0=h4852527_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (dpcpp_impl_linux-64): run-exports library package conda-forge/linux-64::_openmp_mutex==4.5=2_kmp_llvm in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (dpcpp_impl_linux-64): run-exports library package conda-forge/linux-64::libstdcxx-ng==14.2.0=h4852527_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (dpcpp_impl_linux-64): run-exports library package local/linux-64::intel-opencl-rt==2025.0.0=h5a16d59_1169 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

**Pull request merged**